Makefile

```diff
@@ -6,9 +6,9 @@ VPATH = src
 GCOV_OUTPUT = *.gcda *.gcno *.gcov 
 GCOV_CCFLAGS = -fprofile-arcs -ftest-coverage
 SHELL  = /bin/bash
-CFLAGS += -Iinclude -Werror -Werror=return-type -Werror=uninitialized -Wcast-align \
-	  -Wno-pointer-sign -fno-omit-frame-pointer -fno-common -fsigned-char \
-	  -Wunused-variable \
+CFLAGS += -Iinclude -Werror -Werror=return-type -Werror=uninitialized \
+      -Wcast-align -Wshadow -Wno-pointer-sign -fno-omit-frame-pointer \
+	  -fno-common -fsigned-char -Wunused-variable \
```

Comment:
Would you mind simply appending " -Wshadow" on line 11, without cleaning up the other options?